pkg/ring/http.go

```diff
@@ -59,21 +58,32 @@ const tpl = `
 			</table>
 			<br>
 			{{ if .ShowTokens }}
-			<input type="button" value="Hide Ingester Tokens" onclick="window.location.href = '?tokens=false' " />
+			<input type="button" value="Hide Tokens" onclick="window.location.href = '?tokens=false' " />
 			{{ else }}
-			<input type="button" value="Show Ingester Tokens" onclick="window.location.href = '?tokens=true'" />
+			<input type="button" value="Show Tokens" onclick="window.location.href = '?tokens=true'" />
+			{{ end }}
+
+			{{ if .ShowTokens }}
+				{{ range $i, $ing := .Ingesters }}
+					<h2>Instance: {{ .ID }}</h2>
+					Tokens:<br />
+					<ul>
+						{{ range $token := .Tokens }}
+							<li>{{ $token }}</li>
+						{{ end }}
+					</ul>
```

Comment:
I think
```					
<p>
  {{ range $token := .Tokens}}
    {{ $token }}
  {{ end }}
</p>
```

would work fine, it's less markup (easier for browsers to handle), and one doesn't have to scroll huge page. They are still written one per line in HTML, so can be easily diff-ed (I did that a lot when testing memberlist code).

Reply:
The exact markup you posted will end up having all tokens in a single line, not one token per line. Is that what you want? What's the problem listing them like we're doing today?

Reply:
> The exact markup you posted will end up having all tokens in a single line, not one token per line. Is that what you want?

Yes

> What's the problem listing them like we're doing today?

Lot of wasted white space on the screen? :) Nothing wrong, but I don't see any benefit of doing that. [That said, it didn't bother me enough to change it before you started with changes here]

Reply:
I agree. I've changed it to display tokens in a more compact way.